### ISSUE TITLE
Create Profile site - update account

### ISSUE THREAD

**Issue by fingeririna** [AkrosAG/Akros-Marketplace#59]
**Story**
*As a user I want to be able to manage my account information, in order to self-administer my account.*

**Preconditions**
- user is logged in

**Post-Conditions**
- changes are saved in the corresponding database tables

**Out-of-Scope**
- only self-registered users can change their e-mail. Third-party providers are excluded

**Dependencies**
This story must be finished before [Story Nr. 25](https://github.com/AkrosAG/Akros-Marketplace/issues/25) can be started

-----

**DoR**
- [x] External dependencies are identified or eliminated
- [x] Design (at least LoFi?)
- [x] Has acceptance criteria
-----

**Acceptance Criteria**
- [x] the user can change the data under settings/mein profil -> ![Image](https://user-images.githubusercontent.com/27404338/196448451-36ebabc9-d25c-4f9c-ac08-6519a2a4fdd6.PNG)
- [x] the user can change the following data -> ![Image](https://user-images.githubusercontent.com/27404338/196448269-5c6da2cd-6b81-4429-b7ac-e0ddb2a4c7e8.png)
- [x] after hitting the "save" button, the UI should show a text "changes saved!", so that the user has a feedback

-----
**DoD**
- [x] Story is tested against acceptance criteria
- [x] Unit test should be passed
- [x] Integration test is done ( if applicable)
- [x] Non-functional requirements are met
- [ ] Story ok-ed by Product Owner
- [x] Peer Code Review performed
- [ ] Function Review performed
- [x] Any configuration or build changes documented
- [x] any database changes are documented [Database](https://github.com/AkrosAG/Akros-Marketplace/wiki/Database)

**Comment by balsih** [comment]
@TimmyAeberli and @akrguda please verify if ticket is ready for refinement. Any feedback is welcomed!

**Comment by Ayerbe88** [comment]
Profile site has been created accesible form the user menu. ChangeUserData method has been created in user service which calls admin keycloack endpoint with an user ID and a body of the new user data in the form of UserDataModel inteface.

In order to finish the story it is required to export a new keycloack configuration so that new user atribute and required settings are available in new docker images.

New user atributes can be added in keycloack in Realm settings / User profile.

Changing user metadata has only worked from frontend using directly a token generetad in keycloack admin console with the master admin user, this also has to be fixes so all users are able to call the new method ChangeUserData.

usersManagementUrl in environment.prod.ts needs to be changed with correct deployed URL.

User service and profile component habe been excluded from test coverage until task is resolved and no further changes are needed.

**Comment by sereno77** [comment]
Sereno, 05.01.2023 : Testing done, test document done, Test failed

**Comment by LanaKast** [comment]
Tel nr. cannot be saved. Other properties can be saved/updated

**Comment by Atanasov-AA** [comment]
After testing I found out, that the phone number is set in keycloak (this means, that the backend is working), but it's not displayed in the UI, so it's a FE problem.
